### PR TITLE
adds collapsible pledge list view to appeals admin screen

### PR DESCRIPTION
**Pull request by DominikGuzei**
This PR adds the last important feature for the MVP: viewing the pledges for an appeal (admins)
bildschirmfoto 2015-12-10 um 17 32 15
bildschirmfoto 2015-12-10 um 17 32 27

### CODE REVIEW

**Comment by DominikGuzei** [comment]
I just realized that this is our first scenario where we need a migration -> replaying the `AppealsProjection` :wink:

Will add that now

**Comment by DominikGuzei** [comment]
Migration worked well in production -> i have a few private scripts for dumping the production DB + restoring it on the local machine. This is essential for trying out such migrations before pushing them online :wink:

**Comment by rhyslbw** [comment]
Looks great!

File: packages/app/source/server/application.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ Donations.App = Space.Application.define('Donations.App', {
     this.injector.map('Donations.Locations').asStaticValue();
   },
 
+  onStart() {
+    Migrations.migrateTo('latest');
```
Comment (rhyslbw) [inline packages/app/source/server/application.js:37]: This is pooch

File: packages/app/source/server/projections/appeals-projection.js [hunk 2]
```diff
@@ -57,6 +58,17 @@ Space.eventSourcing.Projection.extend(Donations, 'AppealsProjection', {
       $inc: {
         pledgedQuantity: quantity,
         requiredQuantity: -1 * quantity
+      },
```
Comment (rhyslbw) [inline packages/app/source/server/projections/appeals-projection.js:61]: So good

**Pull request merged**